A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/BRK/A320_Neo_BRK.js

```diff
@@ -74,34 +74,26 @@ var A320_Neo_BRK;
             if (this.updateThrottler.canUpdate(_deltaTime) === -1) {
                 return;
             }
-            const currentPKGBrakeState = SimVar.GetSimVarValue("BRAKE PARKING POSITION", "Bool");
+
             const powerAvailable = SimVar.GetSimVarValue("L:DCPowerAvailable","Bool");
             if (this.topGauge != null) {
                 if (powerAvailable) {
-                    this.topGauge.setValue(3);
+                    this.topGauge.setValue(3 * (SimVar.GetSimVarValue("L:A32NX_HYD_BRAKE_ALTN_ACC_PRESS","PSI") / 3000));
                 } else {
                     this.topGauge.setValue(0);
                 }
             }
 
             if (this.leftGauge != null) {
                 if (powerAvailable) {
-                    if (currentPKGBrakeState != 0) {
-                        this.leftGauge.setValue(2);
-                    } else {
-                        this.leftGauge.setValue(2 * (SimVar.GetSimVarValue("BRAKE LEFT POSITION","SINT32") / 32000));
-                    }
+                    this.leftGauge.setValue(3 * (SimVar.GetSimVarValue("L:A32NX_HYD_BRAKE_ALTN_LEFT_PRESS","PSI") / 3000));    
```

Comment:
```suggestion
                    this.leftGauge.setValue(3 * (SimVar.GetSimVarValue("L:A32NX_HYD_BRAKE_ALTN_LEFT_PRESS","PSI") / 3000));
```

Reply:
The PR is still in draft. There are many lint errors to be expected in the Rust code too. I'm sure these will be fixed before the PR comes out of draft 😉.

Reply:
> just simple fix for you.
> if fix this, lint error will disappear :)

Thanks! I wonder why those errors only shows now as there were none just before?

Reply:
Due to EFB. If you have some only 1 error on your code, idk, but EFB seems have lint error. Just fix that, after fix, all error will disappear